app/src/main/java/org/mozilla/webmaker/javascript/WebAppInterface.java

```diff
@@ -299,7 +299,7 @@ public boolean isDebugBuild() {
      * ----------------------------------------
      */
     @JavascriptInterface
-    public String getSystemLocale() {
-        return Locale.getDefault().toString();
+    public String getSystemLanguage() {
+        return Locale.getDefault().toString().replace("_", "-");
```

Comment:
This change was put in place because the browser uses dashes and not underscores like the result currently being outputted.